demos/chat-client/c++/compile_run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# WASI_SDK_PATH defined in https://github.com/WebAssembly/wasi-sdk
+$WASI_SDK_PATH/bin/clang++ main.cpp --sysroot $WASI_SDK_PATH/share/wasi-sysroot -o main.wasm && 
```

Comment:
I don't think this belongs in the repository, looks like a local development environment tooling